File: src/JavaScript/easy/array/problem1389.js
/**
 * @param {number[]} nums
 * @param {number[]} index
 * @return {number[]}
 */


var createTargetArray = function(nums, index) {
    let target = []
    nums.forEach((num, idx) => {
        target.splice(index[idx], 0, num)
    })
    return target
};

let nums = [0,1,2,3,4], index = [0,1,2,2,1]

createTargetArray(nums, index)

File: src/Python/easy/array/problem1389.py
from typing import List

class Solution:
    def createTargetArray(self, nums: List[int], index: List[int]) -> List[int]:
        target = []
        for idx, num in enumerate(nums):
            target.insert(index[idx], num)
        return target

nums = [0,1,2,3,4]; index = [0,1,2,2,1]

solution = Solution()
solution.createTargetArray(nums, index)